docs/package.json

```diff
@@ -38,17 +68,7 @@
       "last 1 safari version"
     ]
   },
-  "devDependencies": {
-    "@babel/core": "^7.21.0",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/preset-typescript": "^7.21.0",
-    "@docusaurus/module-type-aliases": "^2.3.1",
-    "@tsconfig/docusaurus": "^1.0.6",
-    "@types/jest": "^29.4.0",
-    "babel-jest": "^29.4.3",
-    "eslint": "^8.21.0",
-    "eslint-plugin-mdx": "^2.0.2",
-    "jest": "^29.4.3",
-    "typescript": "^4.7.4"
+  "engines": {
+    "node": ">=16.14"
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Eh, fixed in [20cefaf](https://github.com/software-mansion/react-native-reanimated/pull/4904/commits/20cefaf4c132bb368deeb1653ab168a19d50a25b)